src/pages/AdminSettings/AdminMembers/MemberList/MemberTable.test.jsx

```diff
@@ -75,8 +89,9 @@ beforeAll(() => {
 })
 
 beforeEach(() => {
-  server.resetHandlers()
   queryClient.clear()
+  queryClientV5.clear()
```

Comment:
I see we've made a deliberate effort in these suites to modify the ordering of the "clear" statements with the server statement being the last to clear -- jw what the intent was behind this change if any

Reply:
That is the typical flow for the vast majority of our tests so I like the consistency as well as the alphabet, there's no technical reason

Reply:
cool cool, thanks for the context ✅